assignments/arrays.js

```diff
@@ -63,14 +63,14 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a 2011 "jeep" "Wrangler"` );
```

Comment:
The answer is technically correct but you were supposed to reference it using code.
1. Which code do you use to get a specific index value out of an array?
2. Which code do you use to get a specific value out of the property of an object?

The same goes for Challenge 2 below.